### PR TITLE
integration: cleanup apt files

### PR DESCRIPTION
**Pull request by UnmeshDeodhar**
Removing stale apt file for kubernetes.

Fixes: #6666

### CODE REVIEW

**Review by cmaf — APPROVED** [review]
thanks @UnmeshDeodhar

**Comment by fidencio** [comment]
TDX failure is not related to this PR, so, please, go ahead and ignore it.
```
04:14:41 [cleanup_env.sh:55] INFO: Check no kata processes are left behind after reseting kubernetes
04:14:41 [cleanup_env.sh:58] INFO: Checks that pods were not left
04:14:41 make: *** [Makefile:104: cc-kubernetes] Error 1
04:14:41 [run.sh:83] ERROR: sudo -E PATH=/var/lib/jenkins/workspace/tests-CCv0-qemu-TDX-CRI_CONTAINERD_K8S-PR/go/bin:/usr/local/go/bin:/usr/sbin:/sbin:/var/lib/jenkins/workspace/tests-CCv0-qemu-TDX-CRI_CONTAINERD_K8S-PR/go/bin:/usr/local/go/bin:/usr/sbin:/sbin:/usr/sbin:/bin:/usr/bin:/usr/local/bin:/usr/local/sbin CRI_RUNTIME=containerd bash -c make cc-kubernetes
04:14:41 [jenkins_job_build.sh:276] ERROR: .ci/run.sh
04:14:41 ++ handle_error 93
04:14:41 ++ local exit_code=2
04:14:41 ++ local line_number=93
04:14:41 ++ echo 'Failed at /home/jenkins/workspace/tests-CCv0-qemu-TDX-CRI_CONTAINERD_K8S-PR/ci_entry_point.sh +93: .ci/jenkins_job_build.sh "${repo_to_test}" -t "${TEST_TYPE}"'
04:14:41 Failed at /home/jenkins/workspace/tests-CCv0-qemu-TDX-CRI_CONTAINERD_K8S-PR/ci_entry_point.sh +93: .ci/jenkins_job_build.sh "${repo_to_test}" -t "${TEST_TYPE}"
04:14:41 ++ exit 2
```

**Comment by UnmeshDeodhar** [comment]
@wainersm @cmaf @fidencio All tests are passing (except tdx which is unrelated) can we merge this one?

File: integration/kubernetes/cleanup_bare_metal_env.sh [hunk 1]
```diff
@@ -35,6 +35,9 @@ fi
 #cleanup stale file under /run
 sudo sh -c 'rm -rf /run/flannel'
 
+#cleanup the kubernetes apt list file
+sudo rm /etc/apt/sources.list.d/kubernetes.list
```
Comment (wainersm) [inline integration/kubernetes/cleanup_bare_metal_env.sh:39]: @UnmeshDeodhar if `/etc/apt/sources.list.d/kubernetes.list` doesn't exist then this command will fail. Could you add the `-f` flag to ensure it won't fail?

Comment (UnmeshDeodhar) [reply integration/kubernetes/cleanup_bare_metal_env.sh]: Fixed it!

**Review by wainersm — APPROVED** [review]
Thanks @UnmeshDeodhar ! I will wait the tests results to merge it ;)

**Comment by UnmeshDeodhar** [comment]
@wainersm all tests are passing now except TDX. @fidencio mentioned TDX failure is not related to the PR here: https://github.com/kata-containers/tests/pull/5605#issuecomment-1532469864

**Pull request merged**